src/gui/guiFormSpecMenu.cpp

```diff
@@ -2624,7 +2624,7 @@ bool GUIFormSpecMenu::parseStyle(parserData *data, const std::string &element, b
 
 	std::vector<std::string> selectors = split(parts[0], ',');
 	for (size_t sel = 0; sel < selectors.size(); sel++) {
-		std::string selector = trim(selectors[sel]);
+		std::string selector(trim(selectors[sel]));
```

Comment:
In principle a good idea.
But in this case the ugly constructor calls only exist because the rest of the guiFormSpecMenu code is not designed to handle string_views yet.